File: day1/main.go
package main

import (
	"fmt"
	"time"
)

func sumation(a int) int {
	total := 0
	for i := 1; i <= a; i++ {
		total += i
	}
	return total
}

func main() {
	n := 1000000000
	st := time.Now()
	fmt.Println(sumation(n))
	et := time.Now()

	fmt.Println("Time taken: ", et.Sub(st))
}

File: day1/main.py
from time import time

def sumation(n):
    total = 0
    for i in range(n+1):
        total += i
    return total

def main():
    n=1000000000
    st = time()
    print(sumation(n))
    et = time()
    print("Time taken: ", et-st, "sec")

if __name__ == "__main__":
    main()